dokan-wpml.php

```diff
@@ -293,25 +309,30 @@ public function force_redirect_page( $flag, $page_id ) {
      * @since 1.0.0
      *
      * @param  string $language
+     * @param  string $name
      *
      * @return string [$url]
      */
-    public function get_dokan_url_for_language( $language ) {
+    public function get_dokan_url_for_language( $language, $name = '' ) {
         $post_id      = $this->get_raw_option( 'dashboard', 'dokan_pages' );
         $lang_post_id = '';
 
         if ( function_exists( 'wpml_object_id_filter' ) ) {
             $lang_post_id = wpml_object_id_filter( $post_id , 'page', true, $language );
         }
 
-        $url = "";
-
         if ( $lang_post_id != 0 ) {
             $url = get_permalink( $lang_post_id );
         } else {
             $url = apply_filters( 'wpml_home_url', get_option( 'home' ) );
         }
 
+        if ( $name ) {
```

Comment:
@strategio I think you can change this codeblock with newly created function on dokan lite

Reply:
@nurul-umbhiya, I would generally do this if I can be sure this function exists on Dokan Lite.

In WPML we have some dependencies check to force the user to run consistent versions across different addons.

I have no idea if this is something done in Dokan. If not, I would stay on the safe side. This can be replaced later.